content/source/docs/enterprise/admin/upgrades.html.md

```diff
@@ -26,3 +51,37 @@ version. [Learn more about availability during upgrades here](../system-overview
     "Check Now" button; the new version should be recognized.
 1. Click "View Update".
 1. Review the release notes and then click "Install Update".
+
+### Replicated Command Line Interface
+
+1. Login to a command line terminal of the Terraform Enterprise host machine.
```

Comment:
As mentioned above, this feels like a personal preference, but I figured I'd suggest anyways.

```suggestion
1. Connect to the Terraform Enterprise host machine using SSH.
```